src/pages/workspace/upgrade/UpgradeIntro.tsx

```diff
@@ -46,7 +46,16 @@ function UpgradeIntro({feature, onUpgrade, buttonDisabled, loading, isCategorizi
         return `${convertToShortDisplayString(upgradePrice, upgradeCurrency)} `;
     }, [preferredCurrency, isCategorizing]);
 
-    if (!feature || !policyID) {
+    /**
+     *
+     * If the feature is null or there is no policyID, it indicates the user is not associated with any specific workspace.
+     * In this case, the generic upgrade view should be shown.
+     * However, the policyID check is only necessary when the user is not coming from the "Categorize" option.
+     * The "isCategorizing" flag is set to true when the user accesses the "Categorize" option in the Self-DM whisper.
+     * In such scenarios, a separate Categories upgrade UI is displayed.
+     *
+     */
```

Comment:
```suggestion
    /**
     * If the feature is null or there is no policyID, it indicates the user is not associated with any specific workspace.
     * In this case, the generic upgrade view should be shown.
     * However, the policyID check is only necessary when the user is not coming from the "Categorize" option.
     * The "isCategorizing" flag is set to true when the user accesses the "Categorize" option in the Self-DM whisper.
     * In such scenarios, a separate Categories upgrade UI is displayed.
     */
```

Reply:
Love the comment, but i think we should remove the empty lines at the top and bottom!

Reply:
Great, I’ve removed the empty lines.